event/doc.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+// Package event is a library for monitoring events from the MongoDB Go
+// driver. Monitors can be set for commands sent to the MongoDB server and/or
+// changes on the MongoDB server.
+//
+// Monitoring commands requires setting a CommandMonitor object on the
```

Comment:
We should also mention somewhere that `CommandStartedEvent` and the corresponding `CommandSucceededEvent`/`CommandFailedEvent` can be correlated via the `RequestID` fields.